src/main/java/frc/robot/Drivetrain.java

```diff
@@ -28,8 +28,8 @@
      * 
      * @param speed the speed at which to drive (ranges from -1.0 to +1.0)
      */
-    public void driveLeft(double speed){
-        double sp = deadband(speed);
+    public void driveLeft(double speed, boolean boost){
```

Comment:
I like the idea of boost, but I think we should move the implementation "up" to avoid making autonomous code have to deal with it. I recommend moving it outside of the class entirely, and only using it in the teleopPeriodic functions.

Reply:
boost is now called from tank drive and arcade drive methods. 

public void tankDrive(double leftDrive, double rightDrive, boolean boost){
        this.driveLeft(boost(leftDrive, boost));
        this.driveRight(boost(rightDrive, boost));
    }